doc/changes/changelog.md

```diff
@@ -1,3 +1,4 @@
 # Changes
 
-* [1.1.0](changes_1.1.0.md)
\ No newline at end of file
+* [1.1.0](changes_1.1.0.md)
+* [1.1.1](changes_1.1.1.md)
```

Comment:
Maybe we can agree here also on the consistency, would it be better to put the recent version on the top?

Reply:
fixed